Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -23,68 +25,89 @@ public protocol PledgeCTAContainerViewViewModelType {
 public final class PledgeCTAContainerViewViewModel: PledgeCTAContainerViewViewModelType,
   PledgeCTAContainerViewViewModelInputs, PledgeCTAContainerViewViewModelOutputs {
   public init() {
-    let projectAndUser = self.projectAndUserProperty.signal.skipNil()
-    let project = projectAndUser.map(first)
-
-    let pledgeState = project
-      .map(pledgeStateButton(project:))
-
-    let backingEvent = projectAndUser
-      .switchMap { project, user in
-        AppEnvironment.current.apiService.fetchBacking(forProject: project, forUser: user)
-          .materialize()
-      }
-    let backing = backingEvent.values()
+    let project = self.projectProperty.signal.skipNil()
+
+    let backing = project.map { $0.personalization.backing }.skipNil()
+
     let projectAndBacking = Signal.combineLatest(project, backing)
-    let stackViewAndSpacerAreHidden = pledgeState.map { $0.stackViewAndSpacerAreHidden }
+
+    let backedProject = projectAndBacking
+      .filter { isTrue($0.0.personalization.isBacking ?? false) }
+      .map(pledgeCTA(project:backing:))
+
+    let nonbackedProject = project
```

Comment:
Just a small detail here: Could you make the `b` capitalized here? That's the swift standard: `nonBackedProject`